src/include/binder/bound_expression.h

```diff
@@ -31,7 +31,7 @@ class BoundExpression {
   BoundExpression() = default;
   virtual ~BoundExpression() = default;
 
-  virtual auto ToString() const -> std::string { return "<invalid>"; };
+  virtual auto ToString() const -> std::string { return ""; };
```

Comment:
Any particular reason for this change?

Reply:
So that students can be less confusing about the binder output. Previously, if we don't have where clause, it will output `<invalid>` as if the clause is not correctly parsed. We should print nothing instead.

Reply:
I'm thinking of remove `invalid` and use `std::optional` / `nullptr` to represent a non-existing branch. Thinking about it...